lib/app/components/atoms/custom_elevated_button.dart

```diff
@@ -17,7 +17,7 @@ class CustomElevatedButton extends StatelessWidget {
   Widget build(BuildContext context) {
     return SizedBox(
       height: 60,
-      width: 280,
+      width: MediaQuery.of(context).size.width * 0.7,
```

Comment:
```suggestion
      width: Get.width * 0.7,
```
`Get.width`でもデバイス幅取れるのでこっちにしましょう

Reply:
@king-kazu39 
ご指摘の通り修正いたしました。